client/components/GradeForm.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+
+export default class GradeForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: '',
+      course: '',
+      grade: ''
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleCancel = this.handleCancel.bind(this);
+  }
+
+  handleChange(event) {
+    if (event.target.name === 'name') {
+      this.setState({ name: event.target.value });
+    }
+    if (event.target.name === 'course') {
+      this.setState({ course: event.target.value });
+    }
+    if (event.target.name === 'grade') {
+      this.setState({ grade: event.target.value });
+    }
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    const newGrade = {
+      name: this.state.name,
+      course: this.state.course,
+      grade: parseInt(this.state.grade)
+    };
+    this.props.onSubmit(newGrade);
+    this.setState({
+      name: '',
+      course: '',
+      grade: ''
+    });
+    document.getElementById('gradeForm').reset();
```

Comment:
Avoid querying the DOM when authoring React components! Fortunately, you already have the element you are looking for: `event.target`